packages/sanity/src/ui-components/tooltip/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const TOOLTIP_DELAY_PROPS = {
```

Comment:
I think we could consider exporting a TooltipDelayGroupProvider with this default prop as fast follow.